proto/akash/resourcevalue.proto

```diff
@@ -0,0 +1,15 @@
+syntax = "proto3";
+package akash;
+
+import "gogoproto/gogo.proto";
+
+option go_package = "github.com/ovrclk/akash/types";
+
+// Unit stores cpu, memory and storage metrics
+message ResourceValue {
+  option (gogoproto.equal) = true;
+  bytes val = 1 [
+    (gogoproto.nullable) = false,
+    (gogoproto.customtype) = "github.com/cosmos/cosmos-sdk/types.Int"
```

Comment:
why wrap this in `ResourceValue` instead of just using a `types.Int` for the field values?